{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -84,6 +87,11 @@ def get_targets_from_recipe():
 model_name = dbutils.jobs.taskValues.get("Train", "model_name", debugValue="")
 model_version = dbutils.jobs.taskValues.get("Train", "model_version", debugValue="")
 
+if model_uri == "" or model_name == "" or model_version == "":
```

Comment:
Is there ever a case where `model_uri` isn't empty but `model_name` or `model_version` are? And if so, do we still want to override the `model_uri`?

Reply:
No. All three will be set from the notebook of the previous job.

Reply:
So in that case it might be cleaner to just have this since that's the only thing we care about here because we overwrite `model_name` and `model_version`
```suggestion
if model_uri == "":
```

Reply:
We get the three fields either all from jobs util or all from widgets. In case of manually running the notebook, we overwrite all three.
Sure. I don't see any difference and issues will be caught by assertions later.